test/ibmq/test_experiment.py

```diff
@@ -406,6 +406,7 @@ def test_get_analysis_results(self):
         results = self.provider.experiment.analysis_results()
         for res in results:
             self.assertTrue(isinstance(res, AnalysisResult))
+            self.assertEqual(res.verified, None)
```

Comment:
```suggestion
            self.assertFalse(res.verified)
```

I would expect this to be `False` by default, maybe it's None because of some of the boolean conditional logic issues?

Reply:
wasn't sure how optional parameters work here, I assumed an optional when not passed would be None

Reply:
But this is listing analysis results, right? So I think we're calling this to list the results:

https://github.com/Qiskit/qiskit-ibmq-provider/blob/2d26bcbe864d2b2038d96885e82cc4cdf5d265ab/qiskit/providers/ibmq/experiment/experimentservice.py#L308

which constructs the AnalysisResult objects here:

https://github.com/Qiskit/qiskit-ibmq-provider/blob/2d26bcbe864d2b2038d96885e82cc4cdf5d265ab/qiskit/providers/ibmq/experiment/experimentservice.py#L362

Which should be passing the `verified` kwarg here:

https://github.com/Qiskit/qiskit-ibmq-provider/blob/2d26bcbe864d2b2038d96885e82cc4cdf5d265ab/qiskit/providers/ibmq/experiment/analysis_result.py#L182

which should set this:

https://github.com/Qiskit/qiskit-ibmq-provider/blob/2d26bcbe864d2b2038d96885e82cc4cdf5d265ab/qiskit/providers/ibmq/experiment/analysis_result.py#L93

And that should be coming back as `false` by default from the REST API (the default value in the DB is `false` so we should never return `null` back in the JSON).

Reply:
I wonder if the `test/ibmq/test_experiment.py:421:21: E1123: Unexpected keyword argument 'verified' in constructor call (unexpected-keyword-arg)` issue is related to this somehow? Is the kwarg not getting set properly?